### ISSUE TITLE
Relation between learning rate and batch size

### ISSUE THREAD

**Issue by jianlong-yuan** [open-mmlab/mmselfsup#73]
I found that there are some differences in the setting of the learning rate between yours and the paper. Why do you want to set it like this?

In byol/r50_bs256_accumulate16_ep300.py
optimizer = dict(type='LARS', lr=4.8, weight_decay=0.000001, momentum=0.9,
                 paramwise_options={
                    '(bn|gn)(\d+)?.(weight|bias)': dict(weight_decay=0., lars_exclude=True),
                    'bias': dict(weight_decay=0., lars_exclude=True)})

However, in the paper:
LearningRate = 0.2 × BatchSize/256

**Comment by scnuhealthy** [comment]
I follow the setting in the ablation study (introduced in Section F). Notice that epoch=300, initial learning rate=0.3 and bs=4096. Though the config indicates bs=256, gradient accumulation is applied so the total bs is 4096.
